docker-images: use managed prom forwarding image (#39722)
sourcegraph/sourcegraph: docker-images/prometheus-gcp/README.md

# `prometheus-gcp`

> This is a patched version of Sourcegraoh's Prometheus image using an alternative base image provided by GCP to allow automatically shipping metrics to the Google Cloud Managed Prometheus Service.

Reviewer: ```suggestion

This is a patched version of [Sourcegraph's Prometheus image](https://docs.sourcegraph.com/dev/background-information/observability/prometheus) using an alternative base image provided by GCP to allow automatically shipping metrics to the Google Cloud Managed Prometheus Service.
```

Doesn't have to be this PR, but would appreciate a brief mention of this variant in https://docs.sourcegraph.com/dev/background-information/observability/prometheus!

Please note: using this image will automatically begin shipping Prometheus metrics to the Google Cloud Managed Prometheus Service and may incur additional cloud hosting costs.

> This image is built by calling the `build.sh` function in [`prometheus`](../prometheus/README.md).

Reviewer: ```suggestion
This image is built by calling the `build.sh` function in [`prometheus`](../prometheus/README.md) with some additional parameters.
```

> https://cloud.google.com/stackdriver/docs/managed-prometheus/setup-unmanaged#run-gmp

Reviewer: ```suggestion
For more details about Google's forked Prometheus server, see: https://cloud.google.com/stackdriver/docs/managed-prometheus/setup-unmanaged#run-gmp
```
